### ISSUE TITLE
Exec, catch error

### ISSUE THREAD

**Issue by vsoch** [singularityhub/singularity-compose#11]
In the case of an exec error, it's rather ugly:

```bash
$ singularity-compose exec app "AHHH"
/.singularity.d/actions/exec: 9: exec: AHHH: not found
Traceback (most recent call last):
  File "/home/vanessa/anaconda3/bin/singularity-compose", line 11, in <module>
    load_entry_point('singularity-compose==0.0.11', 'console_scripts', 'singularity-compose')()
  File "/home/vanessa/anaconda3/lib/python3.7/site-packages/singularity_compose-0.0.11-py3.7.egg/scompose/client/__init__.py", line 206, in start
    main(args=args, parser=parser, extra=extra)
  File "/home/vanessa/anaconda3/lib/python3.7/site-packages/singularity_compose-0.0.11-py3.7.egg/scompose/client/exec.py", line 30, in main
    project.execute(args.name[0], extra)
  File "/home/vanessa/anaconda3/lib/python3.7/site-packages/singularity_compose-0.0.11-py3.7.egg/scompose/project/project.py", line 273, in execute
    sudo=self.sudo):
  File "/home/vanessa/anaconda3/lib/python3.7/site-packages/spython-0.0.67-py3.7.egg/spython/utils/terminal.py", line 120, in stream_command
    raise subprocess.CalledProcessError(return_code, cmd)
subprocess.CalledProcessError: Command '['sudo', 'singularity', 'exec', 'instance://app', 'AHHH']' returned non-zero exit status 127.
```
We need to catch this.

**Comment by vsoch** [comment]
Here is what the fixed version will look like - the user can still see the message (from Singularity) about the problem:

```bash
$ singularity-compose exec app "AHHH"
/.singularity.d/actions/exec: 9: exec: AHHH: not found
ERROR Command had non zero exit status.
```

**Comment by vsoch** [comment]
Fixed with #12
